bin/5.0.x-dev/prepare_project_edition.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 PROJECT_EDITION=$1
-PROJECT_VERSION=$2
+PROJECT_VERSION=5.0.x-dev
```

Comment:
```suggestion
PROJECT_VERSION=4.6.x-dev
```
We need to start with 4.6 first, only then manually update